article/app/controllers/ArticleController.scala

```diff
@@ -34,7 +38,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
   override def renderItem(path: String)(implicit request: RequestHeader): Future[Result] = mapModel(path)(render(_))
 
   private def renderLatestFrom(model: ArticleWithStoryPackage, lastUpdateBlockId: String)(implicit request: RequestHeader) = {
-      val html = withJsoup(BodyCleaner(model.article, model.article.body)) {
+      val html = withJsoup(BodyCleaner(model.article, model.article.body, false)) {
```

Comment:
I'd recommend adding what the parameter of the false value is for readability e.g 
```
val html = withJsoup(BodyCleaner(model.article, model.article.body, isBlah= false)) {
```